core/src/Cabana_AoSoA.hpp

```diff
@@ -319,18 +319,19 @@ class AoSoA
             0 == sizeof(soa_type) % sizeof(member_value_type<M>),
             "Slice stride cannot be calculated for misaligned memory!" );
 
-        member_pointer_type<M> data_ptr =
+        member_pointer_type<M> ptr =
```

Comment:
Is this conditional still needed when using `staticPtr`? Doesn't `staticPtr(nullptr)==nullptr`?

Reply:
true! I'll get rid of that conditional. That was the result of rebasing on master which had a previous bug fix in this area.